docs/tutorials/prepare_data_rhessi.py

```diff
@@ -0,0 +1,284 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
You understood correctly.

I don't see a reason for placing it in a separate repo. It could go into `tools` if we want to centralise scripts like this. But I thought it might be easier to find if it lives alongside the notebook.